application/common/models/User.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace common\models;
+
+use Yii;
+use yii\helpers\ArrayHelper;
+use yii\web\IdentityInterface;
+
+use common\helpers\Utils;
+
+/**
+ * Class User
+ * @package common\models
+ * @method User self::findOne([])
+ */
+class User extends UserBase implements IdentityInterface
+{
+
+    /**
+     * Validation rules, applies User rules before UserBase rules
+     * @return array
+     */
+    public function rules()
+    {
+        return ArrayHelper::merge(
+            [
+                [
+                    ['uuid'], 'default', 'value' => Utils::generateRandomString(),
+                ],
+
+                [
+                    ['created'],'default', 'value' => Utils::getDatetime(),
+                ],
+
+                [
+                    ['email'], 'email'
+                ],
+            ],
+            parent::rules()
+        );
+    }
+
+    /**
+     * Finds an identity by the given ID.
+     *
+     * @param string|integer $id the ID to be looked for
+     * @return IdentityInterface|null the identity object that matches the given ID.
+     */
+    public static function findIdentity($id)
+    {
+        return static::findOne($id);
+    }
+
+    /**
+     * Finds an identity by the given token.
+     * This method is not supported in this app right now but is required by Yii IdentityInterface
+     *
+     * @param string $token the token to be looked for
+     * @return IdentityInterface|null the identity object that matches the given token.
+     */
+    public static function findIdentityByAccessToken($token, $type = null)
+    {
+        return static::findOne($token);
```

Comment:
Should this return `null` (since we're not supporting this method at the moment) instead of using `$token` as though it were an identity's `$id`?